MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -79,17 +84,19 @@ It is also important to note that while this document provides the criteria agai
 * Module descriptor MUST include interface requirements for all consumed APIs (3, 5)
   * _This is not applicable to libraries_
 * Third party dependencies use an Apache 2.0 compatible license (2)
+* In order to ensure reproducible builds, snapshot versions of build-time dependencies should not be referenced.
 * Installation documentation is included (11)
   * -_note: read more at https://github.com/folio-org/mod-search/blob/master/README.md_
   * _This is not applicable to libraries_
 * Personal data form is completed, accurate, and provided as PERSONAL_DATA_DISCLOSURE.md file (6)
   * _This is not applicable to libraries_
 * Sensitive and environment-specific information is not checked into git repository (6)
-* Written in a language and framework from the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page (3, 5)
+* Written in a language and framework from the [officially supported technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page (3, 5)
```

Comment:
@craigmcnally suggested at the 11/8 TC meeting:
> Relax restrictions about OST list as it pertains to first-party frameworks / technologies, because the deadline for accepting new modules into the release happens before we even have feature freeze on those versions.

Reply:
Does the new 'accepted' status mostly deal with this situation? It seems like probably? If someone submits a TCR with something not yet in 'accepted', the TCR itself might prompt TC to update the list?